apps/internal/oauth/ops/accesstokens/accesstokens_test.go

```diff
@@ -757,18 +757,66 @@ func TestTokenResponseUnmarshal(t *testing.T) {
 			jwtDecoder: jwtDecoderFake,
 		},
 		{
-			desc: "Success",
+			desc: "Success with same expires_in and ext_expires_in",
 			payload: `
 				{
 					"access_token": "secret",
 					"expires_in": 86399,
+					"refresh_in": 43199,
 					"ext_expires_in": 86399,
 					"client_info": {"uid":  "uid","utid": "utid"},
 					"scope": "openid profile"
 				}`,
 			want: TokenResponse{
 				AccessToken:   "secret",
 				ExpiresOn:     internalTime.DurationTime{T: time.Unix(86399, 0)},
+				RefreshOn:     internalTime.DurationTime{T: time.Unix(43199, 0)},
+				ExtExpiresOn:  internalTime.DurationTime{T: time.Unix(86399, 0)},
+				GrantedScopes: Scopes{Slice: []string{"openid", "profile"}},
+				ClientInfo: ClientInfo{
+					UID:  "uid",
+					UTID: "utid",
+				},
+			},
+			jwtDecoder: jwtDecoderFake,
+		},
+		{
+			desc: "Success with different expires_in and refresh On",
+			payload: `
+				{
+					"access_token": "secret",
+					"expires_in": 3600,
+					"refresh_in": 43199,
```

Comment:
To mimic eSTS behavior, I recommend using expires_in = 24h and refresh_in = 12h in tests.

Reply:
done